website/content/docs/agent/wal-logstore/index.mdx

```diff
@@ -45,4 +47,12 @@ The WAL backend has been tested thoroughly during development:
 
 We are aware of how complex and critical disk-persistence is for your data.
 
-We hope that many users at different scales will try WAL in their environments after upgrading to 1.15 or later and report success or failure so that we can confidently replace BoltDB as the default for new clusters in a future release.
\ No newline at end of file
+We hope that many users at different scales will try WAL in their environments after upgrading to 1.15 or later and report success or failure so that we can confidently replace BoltDB as the default for new clusters in a future release.
+
+## Known Issues
+
+The following issues were discovered after release of Consul 1.15.1. Fixes should be available in the next patch release.
+
+ * A follower that is disconnected for a while may be unable to catch up if it is using the WAL backend.
+ * Restoring user snapshots can break replication to WAL-enabled followers.
+ * Restoring user snapshots can cause a WAL-enabled leader to panic.
```

Comment:
```suggestion
## Known issues

The following issues were discovered after releasing Consul 1.15.1 and will be fixed in the next patch release:

 * A follower that is disconnected for a while may be unable to catch up if it is using the WAL backend.
 * Restoring user snapshots can break replication to WAL-enabled followers.
 * Restoring user snapshots can cause a WAL-enabled leader to panic.
```
The style guidance on headings recently changed. Use sentence-style capitalization for all titles and headings. 
We should also use concrete language, which is why I suggested changing "should" to "will" in the opening paragraph, even though the style guidance prohibits describing future functionality. As an experimental feature, this is probably good enough exception. Can we be more concrete about what we mean by "disconnected for a while"? What is a while? 10 seconds? 10 minutes? 10 days?